docs/restic.md

```diff
@@ -90,6 +114,67 @@ one of the following commands:
     kubectl -n heptio-ark get podvolumebackups -l ark.heptio.com/backup-name=YOUR_BACKUP_NAME -o yaml
     ```
 
+## Restoring
+
+## Details of Backup & Restore Process with Restic
+
+This section describes in detail how backups/restores with Ark and restic work.
+
+### Backups
+
+- look for annotation on pods
+- ensure a restic repo exists for the namespace
+    - check if a ResticRepository CR exists
+    - if not, create it
+    - RR controller will first check for an existing repo, if not found, init a new one
+- create PodVolumeBackup CRs and wait for them to complete:
+    - PVB controller runs in a daemonset
+    - the controller on the Pod's node will process the PVBs for it
+        - uses a hostPath volume of /var/lib/kubelet/pods to be able to access volume data
+        - finds the path of the volume to backup
+        - runs `restic backup` 
+        - sets status to complete, or failed
+- get the restic snapshot ID for each volume, and annotate the pod *in the backup* with the ID
+
+
+### Restores
+
+- look for snapshot ID annotation on pods
+- ensure a restic repo CR exists for the namespace
+    - check if a ResticRepository CR exists
+    - if not, create it
+    - RR controller will first check for an existing repo, if not found, init a new one
+- add an init container to the pod, whose job is to wait for a done-file to appear for each volume
+- create the pod in the Kube API
+- create PodVolumeRestore CRs
+- PVR controller runs in daemonset
+    - controller on the pod's node will process the PVRs for it
+        - uses a hostPath volume of /var/lib/kubelet/pods to be able to access volume data
+        - waits for pod to be running our init container (so the volumes are mounted)
+        - finds the path of the volume to restore
+        - runs `restic restore`
+        - writes a done file into the volume, in .ark/, whose name is the UID of the Ark restore
+        - sets status to complete (or failed)
+- init container in the Pod:
+    - polls for done file in each volume, in .ark/, with name = UID of the Ark restore
+    - completes
+
+
+## Limitations
```

Comment:
I'd move this section and Troubleshooting above the section with the technical details.